nix/default.nix

```
@@ -98,8 +100,10 @@ in stdenv.mkDerivation rec {
   '';
 
   installPhase = ''
-    mkdir -p $out/build/android
-    cp -r ./build/android/* $out/build/android/
+    mkdir -p $out/jni
+    cp -r ./build/android/${abidir}/* $out/jni/
+    echo '<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.example.mylibrary" />' > $out/jni/AndroidManifest.xml
```

Looks like a place for a variable.
```suggestion
    echo '${androidManifest}' > $out/jni/AndroidManifest.xml
```